src/main/webapp/i18n/de/modelingSubmission.json

```diff
@@ -15,6 +15,7 @@
             "detail": {
                 "title": "Modeling Submission"
             },
+            "view": "Modeling Submission ansehen",
```

Comment:
please add proper translation here. In German we use:
Modeling --> Modellierung
Submission --> Einreichung

Reply:
and please also change all other wrong values in this file

Reply:
I updated the translations as discussed for all the submission types. Instead of the full variant "Modellierungseinreichung" or "Texteinreichung", the German language version now simply uses "Einreichung" since it is simpler and its meaning should be clear from context anyway.